css/style.css

```
@@ -16,4 +17,161 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* header section */
+ header{
+     display: flex;
+     justify-content: space-evenly;
+     align-content: center;
+ }
 
+ header img{
+     height: 2.5em;
+     padding: 1rem ;
+   
+}
+
+/* Nav bar */
+
+
+ nav ul{
+     display: flex;
+     list-style: none;
+     justify-content: space-evenly;
+     width: 50rem;
+     padding: 1rem 0rem;
+
+ }
+
+ nav ul a{
+     text-decoration: none;
+     font-size: 1.5rem;
+     color:rgb(197, 189, 189);
+ }
+
+nav ul a:hover{
```

```suggestion
nav ul a:focus,
nav ul a:hover{
```
When adding hover, we should also add focus so keyboard users can use the `Tab` key to jump to this and also get the same style